### PR TITLE
Staging 10 31 23 edits

### PR DESCRIPTION
**Pull request by Sophie-edgeworx**
## Summary

Brief explanation of the proposed changes.

Link to Shortcut story.

## Motivation

Why are we doing this? What use cases does it support? What is the expected outcome?

## Checks

- [ ] Run `npm run test` locally.
- [ ] Create a Pull Request (follow the [GitHub flow](https://guides.github.com/introduction/flow/)) and [Conventional Commits Specification](https://www.conventionalcommits.org/en/v1.0.0/))
- [ ] Verify any structural/CSS changes on all screen sizes (if relevant)

### CODE REVIEW

File: content/en/docs/cloud/_index.md [hunk 1]
```diff
@@ -28,55 +28,57 @@ overlay [virtual application network](https://netprototalk.com/2019/11/12/virtua
 4. **Developer familiarity**: feels like the frameworks and tools you already know and love (Docker,
    Kubernetes, Terraform, GitOps, etc).
 
-Built by developers for developers, we focus on beautifully crafted interfaces (web and cli),
+   <!-- TODO Consider rewording this to also target decision makers?>
```
Comment (Sophie-edgeworx) [inline content/en/docs/cloud/_index.md:31]: This might show up in the inspector.

File: content/en/docs/more/changelog.md [hunk 2]
```diff
@@ -170,8 +170,8 @@ several Edgeworx Cloud components.
 #### Action Required
 
 For users that already have nodes registered with Edgeworx Cloud, it is necesary to re-run your
-project's _Node Install Script_ directly (via SSH) on each of your existing nodes. In effect, the
-_Node Install Script_ is performing an upgrade of Edgeworx components on your node. You should have
+project's _Node register Script_ directly (via SSH) on each of your existing nodes. In effect, the
```
Comment (Sophie-edgeworx) [inline content/en/docs/more/changelog.md:173]: Nit: capital letter Register?

File: content/en/docs/more/changelog.md [hunk 3]
```diff
@@ -170,8 +170,8 @@ several Edgeworx Cloud components.
...
+_Node register Script_ is performing an upgrade of Edgeworx components on your node. You should have
```
Comment (Sophie-edgeworx) [inline content/en/docs/more/changelog.md:174]: Nit: capital letter Register?

File: content/en/docs/cloud/_index.md (hunk 1 above)
Comment (ag-edge) [reply content/en/docs/cloud/_index.md:31]: Comments? @neilotoole doe comments show up in browser post linting/compiling.

File: content/en/docs/more/changelog.md (hunk 3 above)
Comment (ag-edge) [reply content/en/docs/more/changelog.md]: Done!

File: content/en/docs/cloud/_index.md (hunk 1 above)
Comment (ag-edge) [reply content/en/docs/cloud/_index.md]: resolved by removing all comments of mine.

File: content/en/docs/apps/microservices.md [hunk 4]
```diff
@@ -60,6 +60,4 @@ different pieces of putting together a Docker container image. {{</info>}}
 
 ## Connecting microservices to edge application
 
-Once your microservice is deployed on your node, you're ready to connect it to your Application!
-
-[//]: # 'Check out the [Port Broker]({{<ref "port-broker.md">}}) section for configuration steps.'
+Once you deploy your one or more microservices, you're ready to connect it to your Application!
```
Comment (ag-edge) [inline content/en/docs/apps/microservices.md:63]: Connect THEM. Fixing and committing

File: content/en/docs/cloud/_index.md [hunk 5]
```diff
@@ -28,55 +28,55 @@ overlay [virtual application network](https://netprototalk.com/2019/11/12/virtua
...
-Terraform or kubernetes then you'll be immediately at home with Edgeworx Cloud.
+Terraform or Kubernetes then you'll be immediately at home with Edgeworx Cloud.
 
 ![Projects Overview](/images/portal-projects-overivew-luckyspin.jpg)
 
 ### Features
 
 - **Edge Native Abstractions**: edge native implementations for agents, agent runtime, volumes,
-  message routes and applications `Kinds`. All driven by Kubernetes
+  message routes and applications `Kinds`. All driven by Kubernetes.
```
Comment (ag-edge) [inline content/en/docs/cloud/_index.md:41]: Period added by mistake.

File: content/en/docs/cloud/_index.md [hunk 6]
```diff
@@ -28,10 +28,10 @@ overlay [virtual application network](https://netprototalk.com/2019/11/12/virtua
 4. **Developer familiarity**: feels like the frameworks and tools you already know and love (Docker,
    Kubernetes, Terraform, GitOps, etc).
 
-Built by developers for developers, we focus on beautifully crafted interfaces (web and cli),
+Built by developers for developers, we focus on functional interfaces (web and CLI),
```
Comment (brain2626) [inline content/en/docs/cloud/_index.md:31]: “functional” is a low bar (and sounds a bit boring) I like “…beautifully crafted interfaces…” but we could come up with some other options if you hate it.

File: content/en/docs/cloud/adding-nodes/add-node.md [hunk 7]
```diff
@@ -4,38 +4,38 @@ weight: 400
...
 
 ![Register Node Script](/images/add-edge-node.png)
 
 Click the `COPY` button to copy the install command to your clipboard.
 
-### Run the node Install script
+### Run the node register script
 
-SSH onto your host (or log in via the Cloud Portal) with a user that is in the `sudo` group.
+SSH into the device using a user with sudo privileges or SSH via Cloud Portal.
```
Comment (brain2626) [inline content/en/docs/cloud/adding-nodes/add-node.md:36]: awkward ..using a user...

File: content/en/docs/cloud/adding-nodes/add-node.md [hunk 8]
```diff
@@ -4,38 +4,38 @@ weight: 400
...
 
-Paste the command line that you copied in the previous step into your terminal.
+Paste the install script that you copied in the previous step into your terminal.
```
Comment (brain2626) [inline content/en/docs/cloud/adding-nodes/add-node.md:38]: node registration script?

File: content/en/docs/cloud/adding-nodes/add-node.md [hunk 9]
```diff
@@ -44,76 +44,52 @@ speed, and other dependencies).
...
+Get the node register script using the below command:
 
 ```bash
-$ edgectl get node-register-script -t
-curl -s https://ecj_vfsw9wess5kheafxaeaeh2kskkv301e9f0ip@api.edgeworx.io/v1/project/b75676cb-ae3f-4808-a992-0811e077d783/node-install-script | sudo bash
+edgectl get node-register-script
 ```
 
-We can use this shell snippet on our edge devices to install Edgeworx Agent and connect to our Edgeworx
+We can use the output of that command on our edge devices to install Edgeworx Agent and connect to your Edgeworx
```
Comment (brain2626) [inline content/en/docs/cloud/adding-nodes/add-node.md:77]: "...on our edge devices..."

"...on your edge devices..."?

File: content/en/docs/cloud/adding-nodes/virtual-node.md [hunk 10]
```diff
@@ -3,12 +3,12 @@ title: "Add a Virtual Node"
 description: "Add a Virtual Node (lightweight local VM) to Edgeworx Cloud"
 weight: 450
 ---
-<!-- TODO: Redo screenshots -->
-Typically you want to push your apps to a "real edge board".
-However, if you do not have an edge board to test on, you can simulate one using the Virtual Node feature.
+
+Typically you want to push your apps to a "real edge node".
+However, if you do not have a device to test on, you can simulate one using the Virtual Node feature.
```
Comment (brain2626) [inline content/en/docs/cloud/adding-nodes/virtual-node.md:8]: "...have a node to test..."

File: content/en/docs/cloud/adding-nodes/add-node.md (hunk 7 above)
Comment (ag-edge) [reply content/en/docs/cloud/adding-nodes/add-node.md]: Fixed

**Pull request merged**